### ISSUE TITLE
Demo for Godot Engine?

### ISSUE THREAD

**Issue by KoxaKoxama** [ToolTech/GizmoSDK-Examples#2]
Is it possible to integrate GizmoSDK streaming into the Godot game engine?

**Comment by ToolTech** [comment]
Yes. That could be a viable soultion as the streaming technique is isolated from the actual game engine and only a small code integration package is needed. My guess is that the streaming technique could be integrated in Unity (update soon)/Unreal/Godot and the more legacy imp Gizmo3D

**Comment by KoxaKoxama** [comment]
Can you tell more about Gizmo streaming system? How much of it is open source and is there any proprietary/closed source code there? Can it be used for commercial purposes?

**Comment by ToolTech** [comment]
The streaming tech is divided into one presentation part and one database part. The presentation part is an opensource impl of various engines. We have release a first iteration of that part to unity but inhous we are in the third generation (inhouse = Saab). That code will soon be release. We also have plans for Unreal, Godot, CryEngine etc.

The database part is a high performance streamed database with infinite storage capacity implemented on any number of local databases and this part i closed source. 

There is a third part as well wich is the content generator wich is a product Saab will relase to customers but we will also make the database stream compatible with open standards (GeoTiff,GDAL,Shape,OpenStreet,3DTiles, etc..)

The code is available for commersial use and for smaller educational projects and indi productions it will be free but for larger sites and apps we would like some payback into our dev efforts but on a low cost basis

**Comment by ToolTech** [comment]
No we Will not go opensource on that APIBest regards!/Anders Modén14 nov. 2023 kl. 18:01 skrev KoxaKoxama ***@***.***>:﻿
Thanks for the clarification.
In the future there will be no plans to open source the code that controls streaming?
